index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <link href="css/all.min.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo-name">
+            
+            <a href="# " >Bahadur</a>
+        </div>
+        <div class="navbar ">
+           
+            <ul class="nav-list">
+               
+               
+                <!-- <li><a href="# ">Portfolio</a></li>
+                <li><a href="# ">About</a></li>
+                <li><a href="# ">Contact Us</a></li>
+                <li><a href="# "><i class="fas fa-envelope"></i></a></li> -->
+                <li > <i class="fas fa-bars fa-2x" ></i></li>
+               
+            </ul> 
+        </div>
+    </header>
+
+    <section class="landing">
+        <div class="about-us">
+            <div class="social-icon">
+                <ul>
+                    <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
+                    <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
+                    <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
+                    <li><a href="#"><i class="fab fa-youtube "></i></a></li>
+                    <li><a href="#"><i class="fab fa-medium "></i></a></li>
+                </ul>
+            </div>
+            <div class="intro ">
+                <h1>Hey there. <br>I am ُSamiullah Bahadur</h1>
+                <h2>I am a Software Developer</h2>
+                <p>I can help you build a product , feature or website Look 
+                    through some of my work and experience! If you like what
+                     you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+        </div>
+        <div class="landing-image ">
+            <img  src="image/landing-origin.png" alt="landing-image "> 
+        </div>
```

Comment:
- [ ] Let's make sure we use any image for styling purposes as a background image  in CSS we should only use images where we need it as part of content, Also, let's make sure we use the same background image as used in Figma 
![image](https://user-images.githubusercontent.com/87689547/149127362-3d424680-3163-4ab8-9221-c47c744b8ca9.png)